.github/workflows/checks-all.yml

```diff
@@ -253,6 +253,26 @@ jobs:
         sudo cp target/debug/movement /usr/local/bin/
         cd -
 
+    - name: Kill running movement process
+      run: |
+        PID=$(ps aux | grep 'movement node run-local-testnet' | grep -v grep | awk '{print $2}' | head -n 1)
+        if [ -n "$PID" ]; then
+          echo "Killing process with PID: $PID"
+          kill -9 $PID
+        else
+          echo "No movement process found."
+        fi
+
+    - name: Delete .movement directory if present
+      run: |
+        cd $GITHUB_WORKSPACE
+        if [ -d "protocol-units/bridge/integration-tests/.movement" ]; then
+          echo ".movement directory found, deleting it."
+          rm -rf protocol-units/bridge/integration-tests/.movement
+        else
+          echo ".movement directory not found."
+        fi
+
```

Comment:
Ah, Now I'm looking at this. I wouldn't run execute this logic from the CI level but rather in the rust integration test logic itself, we want to keep the .yml files clean.

Reply:
Okay I'll close this and add the logic into #523.